### PR TITLE
Users/asthasharma/buildfix

### PR DESCRIPTION
**Pull request by astha-21**
Successful Run : https://github.com/microsoft/query-builder-for-digital-twins/actions/runs/10572096281

### CODE REVIEW

File: QueryBuilder/QueryBuilder.csproj [hunk 1]
```diff
@@ -31,8 +31,8 @@
         <PackageReference Include="System.Text.Encodings.Web" Version="6.0.0" />
     </ItemGroup>
 
-    <ItemGroup Condition="'$(TargetFramework)' != 'net6.0'">
-        <PackageReference Include="System.Text.Encodings.Web" Version="5.0.1" />
+    <ItemGroup Condition="'$(TargetFramework)' == 'net8.0'">
```
Comment (Shuvajyoti) [inline QueryBuilder/QueryBuilder.csproj:34]: Apply net 8 and ! net 8 conditions

Comment (astha-21) [reply QueryBuilder/QueryBuilder.csproj:30]: Done

**Review by saurabh-mridul — APPROVED** [review]

**Comment by astha-21** [comment]
> @astha-21 please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree [company="{Microsoft}"]

**Comment by astha-21** [comment]
> @astha-21 please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree [company="{Microsoft}"]

**Review by Shuvajyoti — APPROVED** [review]

**Pull request merged**